index.jsx

```diff
@@ -214,6 +221,9 @@ class MapInteraction extends Component {
   }
 
   onWheel(e) {
+    const {enableZoom} = this.props;
+    if (!enableZoom) return;
+
```

Comment:
The user can also zoom on mobile by pinching. This is handled in `scaleFromMultiTouch`, so we also need to check for the zoom prop there.